README.md

```diff
@@ -1,7 +1,7 @@
 Grand Central Board for the Apple TV
 ====================================
 
-![image](./README/screenshot.png)
+![image](./README/appletv941.png)
```

Comment:
The widgets will leave this repository. 

But I agree if we have new and more attractive screenshot I would love to add here. The one Adam sent had an error displayed.